apps/connector/fyle_integrations_platform_connector/apis/expenses.py

```diff
@@ -0,0 +1,155 @@
+from typing import List
+from datetime import datetime
+
+from dateutil import parser
+
+from .base import Base
+
+
+class Expenses(Base):
+    """Class for Expenses APIs."""
+    SOURCE_ACCOUNT_MAP = {
+        'PERSONAL_CASH_ACCOUNT': 'PERSONAL',
+        'PERSONAL_CORPORATE_CREDIT_CARD_ACCOUNT': 'CCC'
+    }
+
+    def get(self, fund_source: List[str], import_state: str, last_synced_at: datetime=None,
+        filter_negative_expenses: bool=False) -> List[dict]:
+        """
+        Get expenses.
+
+        Args:
+            fund_source (List[str]): Fund source.
+            import_state (str): Import state.
+            last_synced_at (datetime, optional): Last synced at. Defaults to None.
+            filter_negative_expenses (bool, optional): Filter negative expenses. Defaults to False.
+
+        Returns:
+            List[dict]: Response.
+        """
+        all_expenses = []
+
+        query_params = self.__construct_expenses_query_params(fund_source, import_state, last_synced_at)
+        generator = self.connection.list_all(query_params)
+
+        for expense_list in generator:
+            if filter_negative_expenses:
+                expenses = self.__filter_credit_expenses(expense_list)
+            else:
+                expenses = expense_list['data']
+            all_expenses.extend(expenses)
+
+        return self.__construct_expenses_objects(all_expenses)
+
+
+    @staticmethod
+    def __construct_expenses_query_params(fund_source: List[str], import_state: str, updated_at: datetime) -> dict:
```

Comment:
lets not use the term fund source anywhere, it will be confusing later on, we should start using source_account type directly and any fund source conversions and hacks should be in the integrations code instead of platform connector